bench/wtperf/wtperf_opt.i

```diff
@@ -108,6 +108,8 @@ DEF_OPT_AS_UINT32(database_count, 1,
     "number of WiredTiger databases to use. Each database will execute the"
     " workload using a separate home directory and complete set of worker"
     " threads")
+DEF_OPT_AS_BOOL(distribute_modify, 0,
+    "Distribute the changes of modifications across the document for update")
```

Comment:
@sueloverso, Thanks for the review.
I thought of adding the Modify operation as a test, but I want to find out the overhead of finding out the modify vectors from wiredtiger_calc_modify function for cursor modify operation when that function fails and it leads to cursor update operation because of configured limits of the number of modifications or the total size of modification.

Current update operation has the modifications are very small (1 to 2 bytes) and the number of modifications is around 2 to 3 places, these changes are not good candidates for finding out the overhead from wiredtiger_calc_modify function, so I introduced a distributed modify for 16 modifications of around 10% of record size (limits followed by MongoDB).

I am not sure whether the modify operation is correct for the above operation?

Reply:
I think what I suggest below would get you what you want and allow an easy comparison. You could have a "clean" modify workload, and then have one with the `induce_overhead` or `force_update` workload modifier that forces the failures and then compare runs of one with the others (similar to the shrink/grow workloads).

I think what you want to do is the mostly the same but differently specified and it is easier to see in the configuration as a workload.

Reply:
I changed the approach as per the discussion to modify workload. Please do a complete review again as most of the code is changed.